environment.yml

```diff
@@ -109,3 +109,4 @@ dependencies:
   - pip:
     - git+https://github.com/pandas-dev/pydata-sphinx-theme.git@master
     - git+https://github.com/numpy/numpydoc
+    - pyflakes>=2.2.0
```

Comment:
I think this is available from conda?

Reply:
I'll give it a go on the CI, IIRC I had issues locally with conda but later found an issue in the environment. conda list showed 2.2.0 but flake8 was still using 2.1.1

Reply:
conda doesn't like it.